docs/docs/layout-components.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Layout components
+---
+
+In this guide, you'll learn how to create layout components.
+
+### What are layout components?
+
+Layout components are for sections of your site that you want to share across multiple pages. For example, Gatsby sites will commonly have a layout component with a shared header and footer. Other common things to add to layouts are a sidebar and/or navigation menu. On this page for example, the header at the top is part of gatsbyjs.org’s layout component.
+
+Let’s dive in and explore creating layouts.
+
+### Get up and running with Gatsby
```

Comment:
To avoid duplication, let's make this step assume they have a Gatsby site, and send them to the tutorial instructions if they don't / if they need detail on how to do that.

Examples:
- Sourcing from WordPress links to the quickstart guide: https://www.gatsbyjs.org/docs/sourcing-from-wordpress/#setup
- If you specifically want to use hello-world here, this could link to tutorial part zero, where there are instructions to create a hello world site: https://www.gatsbyjs.org/tutorial/part-zero/#-create-a-gatsby-site